assets/src/data/content/writers/html.tsx

```diff
@@ -203,6 +204,10 @@ export class HtmlParser implements WriterImpl {
     return <Dialog dialog={dialog} context={context} />;
   }
 
+  foreign(ctx: WriterContext, next: Next, element: Foreign) {
+    return <span lang={element.lang || ctx.learningLanguage}>{next()}</span>;
```

Comment:
The server-side render includes a `class="foreign"` but this client-side render does not.  I see that there actually isn't a `foreign` CSS class that we define, which is okay, but let's keep the client and server renderers in sync and have them both either render that class name or both not render it.  I'd vote for having them both render it, even it if is unspecified.